src/int.rs

```diff
@@ -41,32 +41,28 @@ pub use marker_traits::Integer;
 /// Type-level signed integers with positive sign.
 #[derive(Eq, PartialEq, Ord, PartialOrd, Clone, Copy, Hash, Debug, Default)]
 pub struct PInt<U: Unsigned + NonZero> {
-    _marker: PhantomData<U>,
+    pub(crate) n: U
 }
```

Comment:
As before, we modify `PInt` such that `U` exists in the value domain. This allows us to remove uses of unsafe that were needed to construct values of `U` out of thin air.